sites/fast-component-explorer/app/fast-components/configs/fast-tree-item.ts

```diff
@@ -1,30 +0,0 @@
-import {
```

Comment:
@janechu are we not showing the items for menu or tree item in the explorer currently? I noticed that menu item doesn't show up either. Is that intentional?

Reply:
Yes, only whole components are shown not parts of them. This is intended for clarity of use. While menu item is technically a component, its intended use as a part of menu makes it so that menu has to give it a context for it to make sense.